query.md

```diff
@@ -0,0 +1,88 @@
+
+5. All the companies that have less than 1000 employees and have been founded before 2005. Order them by the number of employees and limit the search to 10 companies.
+
+   filter: {$and: [{number_of_employees: {$lt: 1000}}, {founded_year: {\$lt: 2005}}]}
+   sort:{number_of_employees: -1}
+   limit: 10
+
+6. All the companies that don't include the `partners` field.
+
+   filter: {partners: {\$size: 0}}
+
+7. All the companies that have a null type of value on the `category_code` field.
+
+   filter: {category_code: {\$type: "null"} }
+
+8. All the companies that have at least 100 employees but less than 1000. Retrieve only the `name` and `number of employees` fields.
+
+   filter: {$and: [{number_of_employees: {$gt: 100}},{number_of_employees: {\$lt: 1000}}]}
+   project: {name: 1},{number_of_employees: 1}
+
+9. Order all the companies by their IPO price in a descending order.
+   sort: {"ipo.valuation_amount": -1}
+
+10. Retrieve the 10 companies with more employees, order by the `number of employees`
+
+    sort: {number_of_employees: -1}
+    limit: 10
+
+11. All the companies founded on the second semester of the year. Limit your search to 1000 companies.
+
+    filter: {founded_month:6 }
+    limit: 1000
+
+12. All the companies founded before 2000 that have an acquisition amount of more than 10.000.000
+
+    filter: {$and: [{founded_year: {$lt: 2000}}, {"acquisitions.price_amount": {\$gt: 10000000}}]}
+
+13. All the companies that have been acquired after 2010, order by the acquisition amount, and retrieve only their `name` and `acquisition` field.
+
+    filter: {"acquisition.acquired_year": {\$gt: 2010}}
+    project: {name: 1},{acquisitions: 1}
+    sort: {"acquisition.price_amount": -1}
+
+14. Order the companies by their `founded year`, retrieving only their `name` and `founded year`.
+
+    project: {name: 1}, {founded_year:1}
+    sort: {founded_year: -1}
+
+15. All the companies that have been founded on the first seven days of the month, including the seventh. Sort them by their `acquisition price` in a descending order. Limit the search to 10 documents.
+    filter: {founded_day: {\$lte: 7}}
+    sort: {"acquisition.price_amount": 1}
+    limit: 10
+
+16. All the companies on the 'web' `category` that have more than 4000 employees. Sort them by the amount of employees in ascending order.
+
+    filter: {$and: [{category_code:"web"},{number_of_employees: {$gt: 4000}}]}
+    sort: {number_of_employees: 1}
+
+17. All the companies whose acquisition amount is more than 10.000.000, and currency is 'EUR'.
+
+    filter: {$and: [{"acquisition.price_amount": {$gt: 10000000}},{"acquisition.price_currency_code": "EUR" }]}
+
+18. All the companies that have been acquired on the first trimester of the year. Limit the search to 10 companies, and retrieve only their `name` and `acquisition` fields.
+
+    filter: {$and: [{founded_month: {$gte: 1}},{founded_month: {\$lte: 3}}]}
+    project: {name: 1},{acquisitions: 1}
```

Comment:
here you don't need the gte 1 for the month as you can't have a 0 month^^just lte 3 is enougth ;)